CONTRIBUTING.md

```diff
@@ -14,13 +14,19 @@ To set up Feed Filter for local development, first install [Poetry][]:
 
 Go to the [Feed Filter repository][] and tap the **Fork** button at top-right. Then clone the source for your fork and add the upstream project as a Git remote:
 
-    git clone https://github.com/YOUR_USERNAME/feed-filter.git
-    cd feed-filter
-    git remote add upstream feed-filter.git
+    mkdir -p ~/Projects/pelican-plugins
+    git clone https://github.com/YOUR_USERNAME/feed-filter.git ~/Projects/pelican-plugins/
+    cd ~/Projects/pelican-plugins/feed-filter
+    git remote add upstream https://github.com/pelican-plugins/feed-filter.git
+
+Clone the Pelican repository:
 
-The last two steps will install the needed dependencies and set up the project:
+    git clone https://github.com/getpelican/pelican.git ~/Projects/
+
+Install the needed dependencies and set up the project:
 
     poetry install
+    pip install U -e ../../pelican
```

Comment:
I think you meant `-U`.

Reply:
Indeed. Nice catch! I removed that flag in favor of another approach.